src/components/Blogs/index.jsx

```diff
@@ -3,24 +3,42 @@ import { Container, Row } from "react-bootstrap";
 import SearchBar from "../SearchBar";
 import BlogCard from "../BlogCard";
 import "./index.scss";
+import i18next from "i18next";
 
-const POSTS_PER_PAGE = 9;
+const POSTS_PER_PAGE = 18;
 const BLOG_API_URL = `https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts?per_page=${POSTS_PER_PAGE}`;
+const EN_REFERENCE_NUMBER = "10221";
```

Comment:
still i dont know what this means. again imagine the scenario, i am a new teammate. use some comments. explain